src/Entity/Cliente.php

```diff
@@ -28,7 +29,7 @@ public function __construct(
         #[ORM\Column(length: 255)]
         private string $nome,
         #[ORM\Column(length: 11)]
-        private string $cpf,
+        private string $cpf
```

Comment:
deixa para não ter o diff, não faz diferença nas versões mais nova do php, mas as antigas quebram,,,
 ```suggestion
        private string $cpf,
```